plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -354,7 +354,7 @@ open class DefaultPageCreator(
             val extensions = (classlikes as List<WithExtraProperties<DClasslike>>).flatMap {
                 it.extra[CallableExtensions]?.extensions
                     ?.filterIsInstance<Documentable>().orEmpty()
-            }
+            }.distinct()
```

Comment:
Given that we'll be comparing documentables, this might cause performance issues described in #2620 

Can we compare just the DRIs?

```suggestion
            }.distinctBy { it.dri }
```